BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.2.0-preview1-03116-01
+2.2.0-preview1-03116-01
```

Comment:
This is currently a placeholder and will be updated with the package number as soon as the buildtools change is on myget.